client/src/api/fetch.ts

```diff
@@ -60,7 +60,7 @@ export const streamFetch = ({
               });
             } else {
               return reject({
-                message: errMsg || '响应过程出现异常~',
+                message: errMsg || 'There is an exception in the response process~',
```

Comment:
The server error prompt can not be translated first, I think about the server error how to implement i18n